app/assets/stylesheets/course/assessment/submission/submissions.scss

```diff
@@ -11,4 +11,14 @@
       padding: 0 3px;
     }
   }
+
+  &.index {
+    .submissions {
+      .no-submission {
+        strong {
+          color: $red;
```

Comment:
[Hound was complaining](https://github.com/Coursemology/coursemology2/pull/1079#discussion-diff-65035731), unless we disable it?